.githooks/scripts/pylint.sample

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# Ensure the modified files conform to the style enforced by the "Style" CI
+# test. You may change the `pylint_cmd` variable to set the path of your pylint
+# executable.
+
+pylint_cmd=pylint
+
+set -euo pipefail
+cd -P -- "$(dirname -- "$0")/../.."  # cd to the project root
+base_branch=$(git rev-parse --abbrev-ref HEAD | egrep -o '(master|(saas-)?\d+\.\d)')
```

Comment:
Forgot to mention that `\d` for digit exists in python re but not in egrep posix ERE.

I think it should be:
`egrep -o '(master|saas-)?[0-9]+\.[0-9]+'`